src/cljs/rems/application.cljs

```diff
@@ -681,6 +695,8 @@
      [application-state application config]
      [:div.mt-3 [applicants-info application]]
      [:div.mt-3 [applied-resources application userid]]
+     (when (get-in application [:application/permissions :see-everything])
```

Comment:
How about `(contains? (:application/permissions application) :see-everything)`? The `get-in` usage assumes that the reader knows that `(:application/permissions application)` is a set and knows what get returns for Clojure's sets.